daemons/compute/api/datamovement.proto

```diff
@@ -47,11 +47,14 @@ service DataMover {
 // NNF Data Mover will perform a copy from source to the destination. Specify dryrun to instantiate
 // a new data movement request where the copy is simulated and not executed
 message DataMovementCreateRequest {
-    string workflow = 3;
-    string namespace = 4;
-    string source = 5;
-    string destination = 6;
-    bool dryrun = 7;
+    message Workflow {
+        string name = 1;
+        string namespace = 2;
+    }
```

Comment:
Couldn't this message be defined outside of the `DataMovementCreateRequest` since it's used in the other messages as well?

Reply:
I suppose it could. You'll see the other *Request messages just refer to it by the nested name so the end result is the same. But your suggestion is better - that way we'd strip out the "Create" from declaring the workflow... defining `DataMovementCreateRequest_Workflow workflow;` doesn't really make sense if you're asking for status.

Reply:
done